src/mpid/ch4/src/ch4_init.c

```diff
@@ -147,6 +147,12 @@ static int choose_netmod(void)
 #error "Thread Granularity:  Invalid"
 #endif
 
+static const char *MPIDI_CH4_mt_model_names[MPIDI_CH4_NUM_MT_MODELS] = {
```

Comment:
Since this array is used only in this file, can you remove its name prefix. `mt_model_names` should be sufficient.

Reply:
Fixed